gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -2007,7 +2007,11 @@ static PyObject* GemRB_Control_QueryText(PyObject * /*self*/, PyObject* args)
 	if (!ctrl) {
 		return NULL;
 	}
-	char* cStr = MBCStringFromString(ctrl->QueryText());
+
+	String wstring = ctrl->QueryText();
+	std::string nstring(wstring.begin(), wstring.end());
+	char * cStr = (char*) ConvertCharEncoding(nstring.c_str(),
```

Comment:
why are we doing a const cast? just change the return type of the function please.